core/app/models/workarea/release/changeset.rb

```diff
@@ -19,6 +19,29 @@ class Changeset
       index('changeset.product_ids' => 1)
       index('original.product_ids' => 1)
       index('releasable_type' => 1, 'releasable_id' => 1)
+      index(updated_at: 1)
+
+      def self.latest(limit = Workarea.config.per_page)
+        order(updated_at: :desc).limit(limit)
+      end
+
+      def self.summary(release_id)
```

Comment:
So why this instead of `Search::AdminSearch.new(q: '*').facets.detect { |f| f.name == 'upcoming_changes' }`? It seems like the results page we'd land you on would more likely match if the summary was based on those results.

Reply:
what you suggested would give us a list of releases, not a list of releasable types. I guess I could do a search for the release id in upcoming_changes and look at the type facet. Hadn't really thought about it.